### ISSUE TITLE
Support for numpy 2.0 and Python 3.13

### ISSUE THREAD

**Issue by thebaptiste** [Ouranosinc/xclim#2022]
### Addressing a Problem?

Hello

Is xclim 0.53.2 supporting : 
- numpy 2.0 ? 2.1 ? 2.2 ?
- Python 3.13 ?

If not, when do you think xclim will support Python3.13 and at least numpy 2.0 ?
Sorry, maybe I should have opened 2 issues, one for numpy 2 and another for Python 3.13.
Regards

### Potential Solution

Support for numpy 2 and Python 3.13 (if not already supported)

### Contribution

- [ ] I would be willing/able to open a Pull Request to contribute this feature.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by thebaptiste** [comment]
numba 0.61.0 and llvmlite 0.44.0 are now available...

**Comment by aulemahal** [comment]
Indeed. And a PR is open #2054. However, there are other issues than numba, issues that we did not see coming. Any help is welcomed.
